symposion/schedule/admin.py

```diff
@@ -103,5 +107,19 @@ def tutorial_max(self, presentation):
         return getattr(presentation.proposal, 'max_attendees', 'N/A')
     tutorial_max.short_description = 'Attendees Max'
 
+    # When a presentation is saved in the admin we send an email to the staff
+    def save_model(self, request, obj, form, change):
+        # First, save the object
+        obj.save()
```

Comment:
Actually (sorry I missed this earlier) instead of calling `obj.save()` here, we should invoke the method we're overriding, so it can do whatever Django needs to do here (which might change in a new release).  E.g. `super(PresentationAdmin, self).save_model(request, obj, form, change)`.